### ISSUE TITLE
I cannot authenticate to Google

### ISSUE THREAD

**Issue by gnustavo** [watermint/toolbox#670]
**Describe the bug**

I'm trying to authenticate to Google but the command fails every time. 

**To Reproduce**

I'm running this in my Ubuntu 20.04:

```
$ rm -rf ~/.toolbox

$ ./tbx services google sheets sheet list      

watermint toolbox 112.8.787
===========================

© 2016-2022 Takayuki Okazaki
Licensed under open source licenses. Use the `license` command for more detail.

Testing network connection...
The command finished: 2.142s

$ echo $?
6
```
I always get exit code 6.

**Expected behavior**

I expected to be directed to my browser where I should be able to complete the Google authentication workflow.

**Screenshots, or log snippets**

[This is a compressed tar archive](https://drive.google.com/file/d/16mGGZWyXpB3KCbX7h_zNFBGSp-AAagkY/view?usp=sharing) of the `~/.toolbox/jobs` directory of one of my attempts.

**Desktop:**
 - OS: Ubuntu 20.04
 - toolbox Version 112.8.787

**Comment by gnustavo** [comment]
Exactly! I have a Jenkins job running every day using tbx and it started failing yesterday too.

I upgraded tbx to its latest version but it didn't solve the problem. Then I started the authentication configuration from scratch and it didn't work. Then I opened this issue.

**Comment by watermint** [comment]
I'm not sure why this happened. But the issue was not reproduced on the latest dev build (v113).
It probably cert expiration @ImportTaste mentioned. I'm going to release with go module updated version.

**Comment by watermint** [comment]
I believe the issue resolved. Please reopen if the issue reproduced @gnustavo @ImportTaste

**Comment by gnustavo** [comment]
Thank you, @watermint . The new release solved the problem for me.

**Comment by nevch1983** [comment]
I am still getting that same error with exit code 6 and my toolbox is up-to-date.
